metaci/testresults/robot_importer.py

```diff
@@ -186,8 +197,20 @@ def parse_test(test, suite, root):
     zero = timedelta(seconds=0)
     setup_time = _robot_duration(setup.find("status")) if setup else zero
     teardown_time = _robot_duration(teardown.find("status")) if teardown else zero
-
-    # I'm not 100% convinced this is what we want. It's great in the
+    start_time = datetime.now().astimezone(None).isoformat()
+    end_time = datetime.now().astimezone(None).isoformat()
+    if setup:
+        start_time = (
+            _parse_robot_time(setup.find("status").attrib["starttime"])
+            .astimezone(None)
```

Comment:
👍 I appreciate you doing the timezone dance here.

Reply:
Hehe never know where you're gonna be 😄